data/transactions/transaction.go

```diff
@@ -344,6 +344,7 @@ func (tx Transaction) WellFormed(spec SpecialAddresses, proto config.ConsensusPa
 			}
 		}
 
+		effectiveEPP := tx.ExtraProgramPages
```

Comment:
call it `maxExtraProgramPages` ?

Reply:
I think effectiveEPP is better because it doesn't always equal to maxExtraProgramPages. when ApplicationID=0, effectiveEPP = tx.ExtraProgramPages